SAT/SATClause.hpp

```diff
@@ -51,6 +51,7 @@ class SATClause
   void setInference(SATInference* val);
 
   void* operator new(size_t,unsigned length);
+  void operator delete(void *);
```

Comment:
This is weird; 32-bit Clang and GCC complains that this function doesn't exist for generating exception-cleanup code, but never actually calls it and I don't see why it would be needed. If we ever actually need an implementation, it should just call `DEALLOC_KNOWN(this, length)`.

Reply:
Complains in a sense of warnings or errors? I did not have any compile-time errors besides failing static asserts. (But produced binary was broken.)

Reply:
Errors, surprisingly: https://godbolt.org/z/9sb819xT8 - try with/without `-m32`.